TODO.md

```diff
@@ -0,0 +1,71 @@
+# TODO plan for 0.8
+
+These may not all be implemented, but are a tentatively roadmap for things to work for in 0.8.  0.8
+is really meant to be a refinement release rather than institute major underlying changes, with the
+investment paying off by cleaning up the code to allow fast and more reliable future features.
+
+## Installer build improvements
+
+Better installers so we can "make windows" or "make macos" and out pops an installer.  Currently
+installers are too manual and we need to automate this.  More specifically:
+- CPack installer for windows (32 and 64-bit).  If possible figure out how to do a "single"
+  installer that can install the right one.
+- macOS build automation (i.e. up to code signing)
+- lokinet GUI package for linux (debs)
```

Comment:
A fully-fledged `lokinet` installer should include the GUI, which is a separate project. I'm not sure how to best handle this...